docs/Release-Notes.md

```diff
@@ -7,133 +7,138 @@ sidebar_label: Release Notes
-- [1.14.1](1.14.1)
-- [1.14.2](1.14.2)
-- [1.14.3](1.14.3)
-- [1.14.4](1.14.4)
-- [1.14.5](1.14.5)
-- [1.14.6](1.14.6)
-- [1.14.7](1.14.7)
-- [1.14.8](1.14.8)
-- [1.14.9](1.14.9)
-- [1.14.11](1.14.10)
-- [1.15.0](1.15.0)
-- [1.15.1](1.15.1)
-- [1.15.2](1.15.2)
-- [1.15.3](1.15.3)
-- [1.15.4](1.15.4)
-- [1.15.4.post1](1.15.4.post1)
-- [1.15.5](1.15.5)
-- [1.16.0](1.16.0)
-- [1.16.0.post1](1.16.0.post1)
-- [1.16.1](1.16.1)
-- [1.16.2](1.16.2)
-- [1.16.3](1.16.3)
-- [1.16.4](1.16.4)
-- [1.17.0](1.17.0)
-- [1.17.1](1.17.1)
-- [1.17.2](1.17.2)
-- [1.17.2.post1](1.17.2.post1)
-- [1.17.3](1.17.3)
-- [1.17.4](1.17.4)
-- [1.17.5](1.17.5)
-- [1.17.6](1.17.6)
-- [1.17.6.post1](1.17.6.post1)
-- [1.17.6.post2](1.17.6.post2)
-- [1.18.0](1.18.0)
-- [1.18.1](1.18.1)
-- [1.18.2](1.18.2)
-- [1.18.3](1.18.3)
-- [1.18.4](1.18.4)
-- [1.18.5](1.18.5)
-- [1.18.5.post1](1.18.5.post1)
-- [1.18.6](1.18.6)
-- [1.18.7](1.18.7)
-- [1.18.8](1.18.8)
-- [1.18.8.post1](1.18.8.post1)
-- [1.18.9](1.18.9)
-- [1.19.0](1.19.0)
-- [1.19.1](1.19.1)
-- [1.19.2](1.19.2)
-- [1.19.3](1.19.3)
-- [1.19.4.post1](1.19.4.post1)
-- [1.19.4.post2](1.19.4.post2)
-- [1.19.4](1.19.4)
-- [1.19.5](1.19.5)
-- [1.19.6](1.19.6)
-- [1.19.7.post1](1.19.7.post1)
-- [1.19.7](1.19.7)
-- [1.19.8](1.19.8)
-- [1.20.0](1.20.0)
-- [1.20.0rc1](1.20.0rc1)
-- [1.20.1.post1](1.20.1.post1)
-- [1.20.1](1.20.1)
-- [1.20.2](1.20.2)
-- [1.20.3](1.20.3)
-- [1.20.3rc1](1.20.3rc1)
-- [1.20.3rc2](1.20.3rc2)
-- [1.20.4.post1](1.20.4.post1)
-- [1.20.4.post2](1.20.4.post2)
-- [1.20.4](1.20.4)
-- [1.20.4rc1](1.20.4rc1)
-- [1.20.4rc2](1.20.4rc2)
-- [1.20.4rc3](1.20.4rc3)
-- [1.20.5](1.20.5)
-- [1.20.6](1.20.6)
-- [1.20.7](1.20.7)
-- [1.20.8](1.20.8)
-- [1.20.9](1.20.9)
-- [1.20.10](1.20.10)
-- [1.21.0.post1](1.21.0.post1)
-- [1.21.0.post2](1.21.0.post2)
-- [1.21.0](1.21.0)
-- [1.21.0rc1](1.21.0rc1)
-- [1.21.0rc2](1.21.0rc2)
-- [1.21.0rc3](1.21.0rc3)
-- [1.21.1](1.21.1)
-- [1.21.2](1.21.2)
-- [1.21.3](1.21.3)
-- [1.21.4](1.21.4)
-- [1.21.5](1.21.5)
-- [1.21.6](1.21.6)
-- [1.21.7](1.21.7)
-- [1.21.8](1.21.8)
-- [1.21.10](1.21.10)
-- [1.21.11](1.21.11)
-- [1.21.12](1.21.12)
-- [1.22.0](1.22.0)
-- [1.22.0rc1](1.22.0rc1)
-- [1.22.0rc2](1.22.0rc2)
-- [1.22.1](1.22.1)
-- [1.22.2](1.22.2)
-- [1.22.3.post1](1.22.3.post1)
-- [1.22.3](1.22.3)
-- [1.22.4](1.22.4)
-- [1.22.5](1.22.5)
-- [1.22.6](1.22.6)
-- [1.22.7](1.22.7)
-- [1.22.8.post1](1.22.8.post1)
-- [1.22.8](1.22.8)
-- [1.23.0](1.23.0)
-- [1.23.0rc1](1.23.0rc1)
-- [1.23.0rc2](1.23.0rc2)
-- [1.23.1](1.23.1)
-- [1.23.2.post1](1.23.2.post1)
-- [1.23.2.post2](1.23.2.post2)
-- [1.23.2](1.23.2)
-- [1.23.3](1.23.3)
-- [1.23.4](1.23.4)
-- [1.23.5.post1](1.23.5.post1)
-- [1.23.5](1.23.5)
-- [1.23.6](1.23.6)
-- [1.23.6.post1](1.23.6.post1)
-- [1.23.7](1.23.7)
-- [1.23.7.post1](1.23.7.post1)
-- [1.23.8](1.23.8)
-- [1.23.9](1.23.9)
-- [1.23.9.post1](1.23.9.post1)
-- [1.23.9.post2](1.23.9.post2)
-- [1.23.9.post3](1.23.9.post3)
-- [1.23.9.post4](1.23.9.post4)
+- [1.13.3](Release-Notes/1.13.3.md)
```

Comment:
For these I thought to just include them recursively? (I am not exactly sure about the syntax, but quite likely this is possible)
This way, for a new release, I just have to add the release-note, but not edit this table of contents.

Reply:
Let's first look what docusaurus has in store, before resorting to generating more content :) I will have a look, I'll report back if I found something.

Reply:
If I understand the [documentation](https://v1.docusaurus.io/docs/en/api-pages) correctly, you could write such a page as a React component. I would instead go ahead and write a jinja2-based generator for a Markdown page, which seems much more fitting in this community and is not much work at all. I will use some changes of this PR as a base.

Reply:
See #26